app/people/cinema_staff.py

```diff
@@ -0,0 +1,12 @@
+class Cleaner:
+    def __init__(
+            self,
+            name: str
+    ) -> None:
+        self.name = name
+
+    def clean_hall(
+            self,
+            hall_number: int
+    ) -> None:
```

Comment:
```suggestion
class Cleaner:
    def __init__(self, name: str) -> None:
        self.name = name

    def clean_hall(self, hall_number: int) -> None:
```
If you can write it in one line, it's better to do so.